readthedocs/vcs_support/backends/hg.py

```diff
@@ -12,8 +12,8 @@ class Backend(BaseVCS):
     supports_branches = True
     fallback_branch = 'default'
 
-    def update(self):
-        super().update()
+    def update(self, identifier=None):
+        super().update(identifier=identifier)
```

Comment:
I'm not sure what you mean? This is an entirely new kwarg that didn't exist before. So the only effect it can have is what we implement in this PR.

But I can turn it into an `__init__` kwarg for the backend to avoid more confusion.